test/SignatureChecker.t.sol

```diff
@@ -288,6 +428,30 @@ contract SignatureCheckerTests is TestUtils {
         assertTrue(_signatureChecker.isValidECDSASignature(account_, digest_, _encodeSignature(v_, r_, s_)));
     }
 
+    function test_eip2098SignatureIsNotMalleable() public {
```

Comment:
I don't think this is needed at all since it only tests a mock contract that isn't part of `src`. Am I missing something?

Reply:
Yes it was to test that our implementation wasn't vulnerable to signature malleability but since it's not the case, we can remove it.

Reply:
https://github.com/MZero-Labs/common/pull/8/commits/b08fe3cd46db0bcd272a1ce76470bca0d5089713